config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -489,3 +489,55 @@ presubmits:
             requests:
               cpu: 1
               memory: "2Gi"
+
+  - name: pull-perf-tests-unit-tests
+    decorate: true
+    always_run: true
+    spec:
+      containers:
+      - image: golang:1:13
+        command:
+        - make
+        args:
+        - test
+  - name: pull-perf-tests-verify-boilerplate
```

Comment:
Instead of having 5 different jobs, can we create verify-all.sh script that will do all of those?

Reply:
That's possible. For this was I was inspired by sig-scheduling plugins repo config. I don't have strong preferences.